src/StateNode.ts

```diff
@@ -2012,7 +2012,7 @@ class StateNode<
       TEvent
     > = mapValues(
       { ...onConfig, ...doneConfig, ...invokeConfig },
-      (value, event) => {
+      (value: TransitionConfig<TContext, TEvent> | string | StateMachine<any, any, any> | undefined, event) => {
```

Comment:
this is sort of a cast from unknown, unfortunately invokeConfig is typed as any atm so had to cast this somewhere as fixing invokeConfig type was not trivial in a short period of time